tdrs-frontend/src/components/Home/Home.jsx

```diff
@@ -171,7 +177,52 @@ function Home() {
             handleChange={handleChange}
             handleBlur={handleBlur}
           />
-          {shouldShowSttComboBox && (
```

Comment:
@jtimpe looks like we are undoing the work implemented [here](https://github.com/raft-tech/TANF-app/pull/1675/files) to display the STT combobox only for those who sign-in via login.gov. When ACF users request access they do not see the combobox because they are associated with the federal government. in the screenshot below s=staging, q = qasp. i included the email that acf user receives which reflects the missing STT option (this is to be addressed in a future ticket). 

![2133p2](https://github.com/raft-tech/TANF-app/assets/63075587/424fc8a6-98e6-4a54-93e5-af06a787f244)
![2133p1](https://github.com/raft-tech/TANF-app/assets/63075587/df1f58f5-3eb5-47e4-8b9d-5b7a072bbb5c)

Reply:
good catch, thank you. i added the conditional around the Jurisdiction Type selection